h2o/h2o.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+
+echo "BEGIN Stage 1 : Install H2O libraries and dependencies"
+apt-get update
+
+
+
+## Set Spark and Sparkling water versions
+H2O_SPARKLING_WATER_VERSION="3.28.0.3-1"
+SPARK_VERSION=$(spark-submit --version 2>&1 | sed -n 's/.*version[[:blank:]]\+\([0-9]\+\.[0-9]\).*/\1/p' | head -n1)
+echo $SPARK_VERSION
+
+
+
+## Download and unzip Sparking water Scala libraries
```

Comment:
Could you refactor the code into helper functions, e.g., install_sparking_water, install_python_packages, configure_spark?

Reply:
Sure. I have made those changes.